### DISCUSSION TITLE
extremely slow fetch times for previously unvisited feeds: is this just me?

### DISCUSSION THREAD

**Discussion by aaevan** [Athou/commafeed#1072] [Q&A]
Hi! I've used and loved commafeed for years but I'm experiencing some severe slowdowns.

When I go to look at a single feed I haven't looked at before, it takes about 30s (I timed 27s, 32s, 33s, 37s, 23s for several different well-known webcomics) to pull the items. Through one refresh on the same browser, it then works very responsively, but 30s is long enough that it's faster to open the original site and view the post there.

I'm using a login through commafeed.com and I have 86 feeds. I'm on windows 11 with firefox but I have the same problem on ubuntu & firefox. I'm noticing that on a different browser through the same login, the feeds I've already clicked on (in the other instance) are instantaneous but new feeds require the same delay. Is the server (perhaps) caching results from each site's feed and the full lookup time is only on the first visit? Is this upfront lookup time server-wide (across accounts) or would a different user on the same server require the same ~30s upfront wait time for the same feed?

Anyway, I'd love to know what's going on and see whether this is an isolated thing (or just me).

**Comment by travelswagger** [comment]
I have no answer to this but want to second this concern. CommaFeed's update is frankly too too slow now and almost unusable. 

It's also missing previous and next buttons that I found critical to my mobile web view; now, I can only swipe to the next feed posts, which is more time consuming than a quick tap of next.
